shotover-proxy/src/transforms/protect/crypto.rs

```diff
@@ -38,21 +38,24 @@ pub async fn encrypt(
         kek_id: sym_key.key_id,
     };
 
-    Ok(SQLValue::SingleQuotedString(serde_json::to_string(
-        &protected,
-    )?))
+    // TODO: investigate using storing as blob instead of text and then use textAsBlob here
+    // We should functionally verify which has better performance before sticking with one.
```

Comment:
+1

Reply:
Are you requesting that I resolve this TODO before we land the PR?

Reply:
no lets raise as an issue and resolve as we are fixing up the crypto stuff later

Reply:
Sounds good, I added it to https://github.com/shotover/shotover-proxy/issues/676